config/jobs/image-pushing/k8s-staging-build-image.yaml

```diff
@@ -0,0 +1,27 @@
+postsubmits:
+  kubernetes/kubernetes:
+    - name: post-kubernetes-push-image-kube-cross
+      cluster: test-infra-trusted
+      annotations:
+        testgrid-dashboards: sig-release-image-pushes, sig-release-master-informing
+        testgrid-alert-email: release-managers@kubernetes.io
+      decorate: true
+      run_if_changed: '^build\/'
```

Comment:
If this is just building cross, we can slim this to `build/build-image/cross`. Things depend on cross, but cross depends on nothing lol